packages/banner/src/BannerPresenter/presenters.js

```diff
@@ -239,8 +244,9 @@ export function InteractionsWrapper({
 export function IconBackground({
   type,
   stylesheet: customStylesheet,
-  className
+  ...otherProps
 }) {
+  const { className } = otherProps;
   const iconClassName = createCustomClassNames(className, "icon-background");
   const iconImageClassName = createCustomClassNames(
```

Comment:
this is minor but on second thought i'm not sure if there needs to be a class name for the icon image. also right now it's a little bit incorrect so might be better to delete it here and also later on

Reply:
As discussed the naming is happening as expected as there is an `icon-button` component in there that is then adding the custom `className`.